gbalink/CMakeLists.txt

```diff
@@ -1,3 +1,4 @@
 include_directories(. ./../Runtime ${ATHENA_INCLUDE_DIR} ${JBUS_INCLUDE_DIR})
-add_executable(gbalink main.cpp)
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -ldl")
```

Comment:
This is for the '#include <dlfcn.h>' in hecl/extern/boo/logvisor/lib/logvisor.cpp, which calls 'dladdr'.
It looks like llvm is known to struggle at linking this ([I found this on stack overflow](https://stackoverflow.com/questions/12993460/why-am-i-getting-undefined-reference-to-dladdr-even-with-ldl-for-this-simpl)).